services/src/modules/logger.ts

```diff
@@ -11,7 +11,7 @@ const devLoggerConfig: pino.LoggerOptions = {
   level: 'trace',
   prettyPrint: {
     colorize: true,
-    translateTime: 'HH:MM:ss',
+    translateTime: 'HH:MM:ss.l',
```

Comment:
Which formatting engine is being used here where `l` correlates to ms?

Reply:
`pino` logger uses https://github.com/felixge/node-dateformat